src/lib/apis/best-selling.api.ts

```
@@ -0,0 +1,22 @@
+      },
+    });
+
+    if (!payload.ok) {
+      throw new Error(`HTTP error! Status: ${payload.status}`);
+    }
+
+    const data = await payload.json();
+    return Array.isArray(data.products) ? data.products : [];
+
+  } catch (error) {
+    console.error('Failed to fetch Best Selling Products:', error);
```

ESLint

[error] Unexpected console statement.

(no-console)